SingularityUI/app/views/taskHealthcheckNotificationSubview.coffee

```diff
@@ -18,12 +18,24 @@ class taskHealthcheckNotificationSubview extends View
         @listenTo @model, 'sync', @render
         @listenTo @pendingDeploys, 'sync', @render
 
+    deployFailureKilledTask: =>
```

Comment:
I was getting errors using the function at first, but it seems to be working now so I'll use the function.